### DISCUSSION TITLE
[@types/needle] wrong property name

### DISCUSSION THREAD

**Discussion by dean-blechman** [DefinitelyTyped/DefinitelyTyped#63478] [Issues with a @types package] [answered]
Correct me if I'm wrong, but I think `follow_set_cookie` should be `follow_set_cookies`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/needle/index.d.ts#L211

[a comment by an automated account is not shown]

**Comment by nikeee** [comment]
You are right. Do you want to file a PR?

**Comment by martin-badin** [comment, marked as answer]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63554
